test/load-generator/state.go

```diff
@@ -493,7 +493,9 @@ func (s *State) respCodeString() string {
 	s.cMu.Lock()
 	list := codes{}
 	for _, v := range s.respCodes {
-		list = append(list, v)
+		// Dereference the respCode for the codes list so we don't data race once we
```

Comment:
This comment doesn't make a lot of sense outside the context of this change. You have to dereference v in order to match the type of the slice. I think it's okay to leave this uncommented. If you feel like it's important to comment somewhere, maybe on the definition of `type codes`?

Reply:
This is feedback for a change in https://github.com/letsencrypt/boulder/pull/3509 - Can you request it there?